.github/workflows/test.yml

```diff
@@ -18,6 +18,8 @@ jobs:
         uses: actions/setup-node@v2.1.5
         with:
           node-version: '16'
+      - run: sudo apt update && sudo apt install gdal-bin -y
+      - run: gdalinfo --version
```

Comment:
good idea

Reply:
I actually move it to the follow up PR, because it is not needed yet